### PR TITLE
limbo: Misc tests

### PR DESCRIPTION
**Pull request by tetsuo-cpp**

### CODE REVIEW

File: limbo/testcases/webpki.py
Comment (tetsuo-cpp) [inline limbo/testcases/webpki.py:1]: From what I've gathered, this isn't being used anymore and is replaced with `limbo/testcases/webpki/*.py`.

File: limbo/testcases/rfc5280.py [hunk 1]
```diff
@@ -1476,6 +1476,27 @@ def serial_number_zero(builder: Builder) -> None:
...
+
+    The EE cert contains a negative serial number, which is disallowed
+    under RFC 5280.
+    """
+
+    cert_path = ASSETS_PATH / "negative_serial.pem"
+    cert = Certificate(x509.load_pem_x509_certificate(cert_path.read_bytes()))
+
+    builder = builder.server_validation().features([Feature.pedantic_serial_number])
+    builder.trusted_certs(cert).peer_certificate(cert).expected_peer_name(
```
Comment (tetsuo-cpp) [inline limbo/testcases/rfc5280.py:1495]: Bit of an odd test but the vector in pyca was just a single cert and not a full chain. It still exercises what we want though.

File: limbo/testcases/webpki.py
Comment (woodruffw) [reply limbo/testcases/webpki.py:1]: Not quite -- there are still some testcases in here that haven't been migrated. So we should keep this file around.

Comment (woodruffw) [reply limbo/testcases/webpki.py:1]: Oh wait, this is `webpki.py` not `webpki/__init__.py`. In that case yes, good catch, thank you!

**Pull request merged**